sass/main.scss

```diff
@@ -99,6 +100,7 @@ img {
 }
 
 .slide_counts {
+  @extend .section;
```

Comment:
I would recommend against using `@extend` for this. It creates really crazy selectors if you're not careful.